plugins/optimization-detective/optimization.php

```diff
@@ -27,10 +27,10 @@
  * @access private
  * @link https://core.trac.wordpress.org/ticket/43258
  *
- * @param string $passthrough Value for the template_include filter which is passed through.
- * @return string Unmodified value of $passthrough.
+ * @param string|mixed $passthrough Value for the template_include filter which is passed through.
```

Comment:
Would be good to include a note here explaining why that's _not_ always a string.

Reply:
We do the `|mixed` type elsewhere for callbacks for filters. But in general the reason is that other plugins may return anything in filter callbacks, so there is no guarantee for the type.

So you want something like the following?

```suggestion
 * @param string|mixed $passthrough Value for the template_include filter which is passed through. (Some plugins may have filtered this to return `null` or any other value type.)
```

Reply:
Ah, nevermind then. I somehow was confused thinking this was a callback to a PHP function like `ob_start()`, but since it's a filter callback, it's fine without a comment.